src/pb_stub.cc

```diff
@@ -1604,6 +1546,17 @@ PYBIND11_EMBEDDED_MODULE(c_python_backend_utils, module)
   module.attr("MetricFamily").attr("COUNTER") = MetricKind::COUNTER;
   module.attr("MetricFamily").attr("GAUGE") = MetricKind::GAUGE;
 
+  module.def(
+      "load_model", &LoadModel, py::arg("model_name").none(false),
+      py::arg("config").none(false) = "",
+      py::arg("files").none(false) = py::dict());
```

Comment:
I think empty dictionary as initializer can have unexpected consequences. Can we use none here instead and do type checking inside `load_model`?

https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python